de_DE/LC_MESSAGES/discordbot.po

```diff
@@ -98,27 +98,25 @@ msgstr "Verwendung"
 # ONLY TRANSLATE: "usage", "commands"
 msgid "USAGE_INTRO"
 msgstr ""
-"Anwendung: .sudo ...\n"
+"Anwendung: .sudo <Aktion>\n"
 "\n"
 "Befehle:"
 
 msgid "USAGE_draw"
 msgstr ""
 "   draw\n"
-"      Draw a Regular Ticket and see if you win a prize!"
+"      Zieh ein normales Ticket. Mit etwas Glück gewinnst du einen Preis."
```

Comment:
Needs exclamation mark and feels a little awkward to split it up into two sentences when the other languages don't.